core/tracker/tracker.go

```diff
@@ -271,15 +271,16 @@ func analyseFetcherFailed(duty core.Duty, allEvents map[core.Duty][]event) (bool
 	msg := msgFetcher
 
 	// Proposer duties depend on randao duty, so check if that was why it failed.
-	if duty.Type == core.DutyProposer || duty.Type == core.DutyBuilderProposer { //nolint:nestif
+	if duty.Type == core.DutyProposer || duty.Type == core.DutyBuilderProposer {
 		// Proposer duties will fail if core.DutyRandao fails
 		randaoFailed, randaoComp := dutyFailedComponent(allEvents[core.NewRandaoDuty(duty.Slot)])
 		if randaoFailed {
-			if randaoComp == parSigDBThreshold {
```

Comment:
this `if` block is 3 layers deep

Reply:
nice hack, switch is being used like an if... 🤣